samples/Templates/Scenarios/ExpenseReport.template.json

```diff
@@ -209,30 +204,31 @@
 						},
 						{
 							"type": "Column",
-							"spacing": "Small",
+							"spacing": "small",
 							"selectAction": {
 								"type": "Action.ToggleVisibility",
+								"title": "expand",
 								"targetElements": [
 									"cardContent${$index}",
 									"chevronDown${$index}",
 									"chevronUp${$index}"
 								]
 							},
-							"verticalContentAlignment": "Center",
+							"verticalContentAlignment": "center",
 							"items": [
 								{
 									"type": "Image",
 									"id": "chevronDown${$index}",
 									"url": "https://adaptivecards.io/content/down.png",
 									"width": "20px",
-									"altText": "${description} $${total} collapsed"
+									"altText": "Details collapsed"
```

Comment:
is this change intentional?

Reply:
This was probably a difference between `ExpenseReport` and `ExpenseReportWithLabels`. Most of the changes in this file occurred because I **deleted** `ExpenseReport` and **renamed** `ExpenseReportWithLabels`